src/Models/Product.php

```diff
@@ -165,10 +160,20 @@ public function getLabel(): string
         return $this->getLabelFromAttribute();
     }
 
+    public function getVariantLabel(): string
```

Comment:
This whole method can be removed, attribute for the variant is implementation specific. It is up to the implementator to fetch the correct attribute when rendering the variant label.